### PR TITLE
[Doc] Add Chinese doc for tutorials05_training_tricks_md

### PR DESCRIPTION
**Pull request by MengzhangLI**
Add Chinese doc for tutorials05_training_tricks_md

### CODE REVIEW

File: docs_zh-CN/tutorials/training_tricks.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# 教程 5: 训练小技巧
+# 教程 5: 训练技巧
+
+MMSegmentation 支持如下训练技巧：
+
+## 骨架和头组件使用不同的学习率 (Learning Rate, LR)
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/training_tricks.md:5]:
```suggestion
## 主干网络和头组件使用不同的学习率 (Learning Rate, LR)
```

File: docs_zh-CN/tutorials/training_tricks.md [hunk 2]
```diff
@@ -1 +1,51 @@
-# 教程 5: 训练小技巧
+# 教程 5: 训练技巧
+
+MMSegmentation 支持如下训练技巧：
+
+## 主干网络和头组件使用不同的学习率 (Learning Rate, LR)
+
+在语义分割里，一些方法会让头组件的学习率大于骨架组件的学习率，这样可以获得更好的表现或更快的收敛。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/training_tricks.md:7]:
```suggestion
在语义分割里，一些方法会让头组件的学习率大于主干网络的学习率，这样可以获得更好的表现或更快的收敛。
```

Comment (xvjiarui) [reply docs_zh-CN/tutorials/training_tricks.md:7]: We may replace them all.

Comment (MengzhangLI) [reply docs_zh-CN/tutorials/training_tricks.md]: > We may replace them all.

OK, I will replace all of it in every single doc.

**Comment by MengzhangLI** [comment]
Just pulled new version, please check it out.

Best,

**Pull request merged**